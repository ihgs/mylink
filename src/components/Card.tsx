import { ReactNode } from "react";

const Card = ({title, children}:{title: string, children: ReactNode}) =>{
  

  return (
    <div className="max-w-sm w-60 my-3 mr-3  rounded overflow-hidden bg-[#fca5a5] shadow-lg text-left px-5 py-4">
        <div className="text-2xl pb-3">{title}</div>
        <div>
          {children}
        </div>
      </div>
  )

}

export default Card;