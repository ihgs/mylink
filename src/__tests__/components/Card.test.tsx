import { render } from "@testing-library/react";
import Card from "../../components/Card";
test("[snapshot]Card", () => {
  const view = render(
    <Card title="test">
      <>
        <ul>
          <li>one</li>
          <li>two</li>
        </ul>
      </>
    </Card>
  );
  expect(view.container).toMatchSnapshot();
});
