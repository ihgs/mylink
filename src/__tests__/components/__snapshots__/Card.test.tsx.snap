// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[snapshot]Card 1`] = `
<div>
  <div
    class="max-w-sm w-60 my-3 mr-3  rounded overflow-hidden bg-[#fca5a5] shadow-lg text-left px-5 py-4"
  >
    <div
      class="text-2xl pb-3"
    >
      test
    </div>
    <div>
      <ul>
        <li>
          one
        </li>
        <li>
          two
        </li>
      </ul>
    </div>
  </div>
</div>
`;
